taskbar.js

```diff
@@ -928,6 +928,8 @@ var Taskbar = class {
             return;
         }
 
+        this.dtpPanel._initProgressManager();
```

Comment:
I don't think it's a good idea to put that in `_redisplay`

Reply:
If I recall correctly, I chose to place it in `_redisplay` because that function is a hinge point in the panel reloading process that happens when changing screens or unlocking the session. I can't remember which issue this line fixed, but if you have a better insight (or better debugging tools) I'm all ears